pyuvdata/utils.py

```diff
@@ -1389,6 +1389,9 @@ def get_baseline_redundancies(baselines, baseline_vecs, tol=1.0, with_conjugates
         vec_bin_centers[gi] = np.mean(baseline_vecs[inds, :], axis=0)
 
     lens = np.sqrt(np.sum(vec_bin_centers**2, axis=1))
+    if np.sum([len(bg) for bg in bl_gps]) > Nbls:
+        raise ValueError("Some baselines are falling into multiple"
+                         " redundant groups. Redundancy tolerance is causing ambiguity.")
```

Comment:
Maybe suggest decreasing the tolerance? It should be clear that's the problem from what you've got but a specific direction about how to fix it would be nice.